### PR TITLE
15 section nav

### LINKED ISSUE #15: SectionNav
## Component Type: Stateless

Check each box once you have completed done the task.

-   [x] Component created
-   [x] Styles written
-   [x] Storybook created, with stories for all normal states
-   [x] Tests written, with snapshot test at minimum
-   [ ] Code reviewed

Add PR link here when ready:

### PR DESCRIPTION
**Pull request by S-unya**
**Ensure each of these tasks is complete before merging**

Use component name as PR title and request code review before merging. You can
open a PR and request review for an incomplete component if you'd like.

-   [x] Component created
-   [x] Styles written
-   [x] Storybook created, with stories for all normal states
-   [x] Tests written, with snapshot test at minimum

Closes #15

### CODE REVIEW

File: src/components/SectionNav/SectionNav.tsx [hunk 1]
```diff
@@ -84,13 +66,16 @@ const createNavItems = (
...
     style,
     className,
-    onNavigation
+    onNavigation,
+    navItemClassName,
+    navWrapperClassName
 }) => (
-    <nav className={[styles.component, className].join(" ")} style={style}>
-        {createNavItems(sections, keyConsts, onNavigation)}
+    <nav className={cx(className)} style={style}>
```
Comment (ascorbic) [inline src/components/SectionNav/SectionNav.tsx]: Any reason to wrap a single classname with className?

Comment (S-unya) [reply src/components/SectionNav/SectionNav.tsx]: I vacillated and decided that having it there in concordance with the convention was appropriate, and there is a possibility that we may want to add some definition styles after all.

Happy to go either way on this though

Comment (S-unya) [reply src/components/SectionNav/SectionNav.tsx]: I think I should convert the name `cx` to `classNames` as elsewhere though

**Comment by ascorbic** [comment]
Storybook build

Built with commit d2705762b87b39c21eac8bfe2ba156039e05e7d6

https://deploy-preview-61--pedantic-bassi-74999b.netlify.com

**Pull request merged**